templates/case-studies/base_case-studies.html

```diff
@@ -0,0 +1,21 @@
+{% extends "templates/base.html" %}
+
+{% block meta_copydoc %}{% endblock meta_copydoc %}
+{% block body_class %}is-paper{% endblock body_class %}
+
+{% block outer_content %}
+  {% block content %}{% endblock %}
+  <section class="p-strip is-deep">
+    <div class="row">
```

Comment:
Looks like a redundant section with no content

Reply:
I added the empty section just for this example to show the CTA footer, this will be removed once the other sections are added to the example.